x/slashing/types/params.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // Default parameter namespace
 const (
-	DefaultSignedBlocksWindow   = int64(100)
+	DefaultSignedBlocksWindow   = int64(5760)
```

Comment:
here we are changing the defaults window from 100 blocks to roughly the number of blocks per day assuming 4 blocks a minute. (24*60*4)

Reply:
How is this parameter used by the Cosmos SDK?

Reply:
its used when slashing for liveness. If a validator misses a certain percentage of signatures in this window, they will be jailed and slashed. Currently its 50%, so validators will be slashed if they miss roughly half the signatures in one day.

the parameter [spec](https://github.com/celestiaorg/cosmos-sdk/blob/release/v0.46.x-celestia/x/slashing/spec/08_params.md) 

[docs](https://docs.cosmos.network/main/modules/slashing#liveness-tracking)

source
https://github.com/celestiaorg/cosmos-sdk/blob/d11c2227e0b34e641b6269627245b206bf67020c/x/slashing/keeper/infractions.go#L72-L87